libs/barista-components/empty-state/src/empty-state.ts

```diff
@@ -267,9 +267,11 @@ export class DtEmptyState
    * @breaking-change Remove with version 8.0.0
    */
   _updateLayout(): void {
-    const componentWidth = this._elementRef.nativeElement.getBoundingClientRect()
-      .width;
-    this._updateLayoutForSize(componentWidth);
+    if (this._platform.isBrowser) {
+      const componentWidth = this._elementRef.nativeElement.getBoundingClientRect()
+        .width;
+      this._updateLayoutForSize(componentWidth);
+    }
```

Comment:
These changes make PR #972 obsolete. I am fine with merging this and closing #972

Reply:
I guess during the review this PR got already merged. Either rebase and use your changeset or remove this changeset from the PR.

Reply:
It is not obsolete because the `_updateLayout()` is currently called from the `_updateLayout` function inside the `DtCustomEmptyStateBase`.  
![image](https://user-images.githubusercontent.com/11156362/81527848-963b5800-935b-11ea-8d5b-1d6d687c2a3e.png)

Reply:
Your PR added only the check that the `_updateLayout` function is not called inside the `ngAfterViewInit`